README.md

```diff
@@ -111,18 +111,9 @@ cd lightdash
 # Install Lightdash dependencies and build
 yarn install
 yarn build
-```
-
-### Launching Lightdash
-
-```shell
-# Specify the path to your dbt project
-# (i.e. the directory containing dbt_project.yml)
-# You MUST use the absolute path (i.e no ../../myrepo)
-export DBT_PROJECT_DIR=/Users/myuser/dbtrepo
-export DBT_PROFILES_DIR=/Users/myuser/.dbt
 
-# Build and run Lightdash
+# Run Lightdash
+export LIGHTDASH_CONFIG_FILE=${PWD}/lightdash.yml
```

Comment:
Probably better to have an example of the lightdash.yml for a dbt project in the README. Otherwise people need to leave github and check the docs website.

Reply:
There's a default in the repo, do you think we should also document how they can edit it?

Reply:
Just having an example would be good. So people get an idea of what is inside.

Reply:
Added this docs/docs/references/lightdashyaml.md